tests/compas/datastructures/test_halfedge.py

```diff
@@ -0,0 +1,95 @@
+import pytest
+
+from compas.datastructures import HalfEdge
+
+
+# ==============================================================================
+# Fixtures
+# ==============================================================================
+
+@pytest.fixture
+def mesh():
+    vertices = [
+        [1.0, 0.0, 0.0],
+        [1.0, 2.0, 0.0],
+        [0.0, 1.0, 0.0],
+        [2.0, 1.0, 0.0]
+    ]
+
+    faces = [
+        [0, 1, 2],
+        [0, 3, 1]
+    ]
+
+    he = HalfEdge()
+
+    for vertex in vertices:
+        he.add_vertex(attr_dict={c: xyz for c, xyz in zip("xyz", vertex)})
```

Comment:
why not just write this out?
tests should be as clear as possible and this is unnecessarily unclear to me...